.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es6": true,
+        "amd": true,
+        "jest": true
+    },
+    "extends": "airbnb",
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+        "rules": {
+
+    }
+};
```

Comment:
Make sure to leave a newline at the end of each file. https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline